src/cli/utils/sharepoint.go

```diff
@@ -36,14 +42,65 @@ type SharePointOpts struct {
 	Populated PopulatedFlags
 }
 
+// AddSharePointDetailsAndRestoreFlags adds flags that are common to both the
+// details and restore commands.
+func AddSharePointDetailsAndRestoreFlags(cmd *cobra.Command) {
+	fs := cmd.Flags()
+
+	fs.StringVar(
+		&Library,
+		LibraryFN, "",
+		"Select only this library. Default includes all libraries.")
```

Comment:
```suggestion
		"Select only this library; defaults to all libraries")
```
Consistency with flag below